### PR TITLE
Unify time modification on SD and SPIFFS

### PR DESCRIPTION
**Pull request by luc-github**
need https://github.com/espressif/esp-idf/pull/1128
to be applied to work

### CODE REVIEW

File: libraries/FS/src/FS.h [hunk 1]
```diff
@@ -70,6 +70,7 @@ class File : public Stream
     size_t size() const;
     void close();
     operator bool() const;
+   time_t getLastWrite();
```
Comment (me-no-dev) [inline libraries/FS/src/FS.h]: missing space here? :)

**Comment by me-no-dev** [comment]
@luc-github this good to go?

File: libraries/FS/src/FS.h (hunk 1 above)
Comment (luc-github) [reply libraries/FS/src/FS.h]: Ho bad formating ...

**Comment by luc-github** [comment]
Need idf first no ?

File: libraries/FS/src/FS.h (hunk 1 above)
Comment (luc-github) [reply libraries/FS/src/FS.h]: Fixed it - thank you

**Comment by luc-github** [comment]
thank you ^_^

**Pull request merged**